wandb/analytics/__init__.py

```diff
@@ -1,3 +1,3 @@
-__all__ = ("Sentry",)
+__all__ = ("get_sentry",)
```

Comment:
Sort of unusual choice we had here; I usually see `__all__` defined after the imports.

Reply:
Oh nice!! I'll do that in my future PRs.